pl-PL.json

```diff
@@ -172,7 +172,7 @@
 	"WRONG_SUBS": "Złe napisy?",
 	"ERR_SUBS_LOADING_TITLE": "Błąd podczas ładowania napisów",
 	"ERR_SUBS_LOADING": "Nie udało się załadować napisów: to może być problem z dodatkiem, którego używasz, lub z twoim połączeniem sieciowym",
-	"WARN_FOLLOWING_SUB_ADDONS_FAILED": "Te dodatki z napisami poległy",
+	"WARN_FOLLOWING_SUB_ADDONS_FAILED": "Następujące dodatki do napisów nie powiodły się",
```

Comment:
This is used in the player view, when we want to display an alert that some addons failed to load subtitles. First version wasnt perfect but this one this one is not correct gramatically. Perhaps something like `Następujące dodatki do napisów nie załadowały się poprawnie`

Reply:
Yeah tbh your version is a lot better. 😅 Will fix that